src/ray/core_worker/transport/raylet_transport.cc

```diff
@@ -49,10 +49,8 @@ void CoreWorkerRayletTaskReceiver::HandleAssignTask(
         /*transport_type=*/static_cast<int>(TaskTransportType::RAYLET));
     Status status = object_interface_.Put(*results[i], id);
     if (!status.ok()) {
-      // TODO (kfstorm): RAY_LOG(FATAL) except the error is about the object to put
-      // already exists.
-      RAY_LOG(WARNING) << "Task " << task_spec.TaskId() << " failed to put object " << id
-                       << " in store: " << status.message();
+      RAY_LOG(FATAL) << "Task " << task_spec.TaskId() << " failed to put object " << id
```

Comment:
would be good to add a comment here that we use log FATAL for put errors except when object exists.

Reply:
good point.